src/pages/resources.mdx

```diff
@@ -0,0 +1,35 @@
+import HeroBanner from '@site/src/components/HeroBanner'
+
+<HeroBanner title="Resources" />
+
+Welcome to the Fastify resources page! Here, you can find a collection of useful resources for learning and exploring Fastify.
+
+## Books
+
+If you're looking for an in-depth resource to learn Fastify, consider this book:
+
+- [Accelerating Server-Side Development with Fastify: A comprehensive guide to API development for building a scalable backend for your web apps](https://www.amazon.it/Accelerating-Server-Side-Development-Fastify-comprehensive/dp/1800563582/ref=sr_1_1?__mk_it_IT=%C3%85M%C3%85%C5%BD%C3%95%C3%91&crid=3V6FTJ14DL1VN&keywords=fastify&qid=1697134067&sprefix=fastify%2Caps%2C258&sr=8-1)
+
+## Talks
+
+Watch these talks to learn more about Fastify:
+
+- [A "fast" Introduction to Fastify - Matteo Collina, NearForm](https://www.youtube.com/watch?v=MOvLdfUhqrU&list=PLAv2aQ9JgGbVAGY0ht6JMoGJ8tF_MJ4fx&index=4)
```

Comment:
You basically linked the fourth video of the playlist.

Maybe should be linked to each specific video.